libraries/backend/azure_connection.rb

```diff
@@ -98,7 +98,7 @@ def rest_get_call(url, params = {})
     headers = {}
     headers['User-Agent'] = INSPEC_USER_AGENT
     headers['Authorization'] = "#{@@token_data[resource.to_sym][:token_type]} #{@@token_data[resource.to_sym][:token]}"
-    headers['Content-Type'] = 'application/json'
+    headers['Accept'] = 'application/json'
```

Comment:
Is this a change in the graph API headers? If so, is there a scenario where `Content-Type` is still required?